doc/user/layouts/partials/create-source/connector/s3/details.html

```diff
@@ -58,18 +56,34 @@
   SQS NOTIFICATIONS 'example-notifications'
 ```
 
+[object-specifiers]: https://github.com/MaterializeInc/materialize/issues/5502
+
+##### Listing Bucket objects
+
+The `BUCKET SCAN` discovery performs a single scan over the specified bucket at source creation
+time. If you would like an S3 source to ingest objects that are added to the bucket after the
+source is created you must also configure an `SQS NOTIFICATIONS` discovery mechanism on the source.
+
 ##### Listening to SQS notifications
 
 AWS S3 has a built-in method for notifying downstream applications of bucket modification, the
-[Event Notification API][notification-aws]. For Materialize, the only interesting
-modifications are object creation, aka the `s3:ObjectCreated:*` event namespace. Follow [the
-AWS Tutorial][notification-tutorial] to configure a bucket for exactly this namespace.
+[Event Notification API][notification-aws]. For Materialize, the only interesting modifications are
+object creation, aka the `s3:ObjectCreated:*` event namespace. Follow [the AWS
+Tutorial][notification-tutorial] to configure a bucket for exactly this namespace.
 
 Once you have configured S3 notifications to go to an SQS queue, you can point Materialize at that
 queue with **DISCOVER OBJECTS USING SQS NOTIFICATIONS `'queue-name'`**.
 
+Materialize deletes SQS messages as soon as they are ingested -- the same SQS queue cannot be used
+for multiple sources. If you would like to have multiple Materialize sources listening to
```

Comment:
I would guess from this that having an unmaterialized s3 source with multiple materialized views on it would act the same here, and each view would see a different subset of messages? If so, that should be called out here too. We might also consider having the sql layer error in that case, because it's never going to be correct.

Reply:
Good call, you're correct until we have the create view in transaction feature.